lib/iterator.js

```diff
@@ -328,14 +348,30 @@ class Iterator {
     return iter(res && res[Symbol.iterator] ? res : [res]);
   }
 
-  // Create iterator by zipping multiple provided iterators into one
-  // Signature: ...iterators
+  // Creates an iterator that aggregates elements from each of the iterators.
+  //
+  // Returns an iterator of Arrays where the i-th tuple contains
+  // the i-th element from each of the passed iterators.
+  // The iterator stops when the shortest input iterable is exhausted.
+  // Signature: base, ...iterators
```

Comment:
It was written this way to specifically hide the implementation detail of having a separate argument which has the same meaning as every other
```suggestion
  // Signature: ...iterators
```